test/jdk/java/lang/StringBuffer/IndexOf.java

```diff
@@ -31,158 +31,280 @@
 
 public class IndexOf {
 
-    static Random generator = new Random();
-    private static boolean failure = false;
+  static Random generator = new Random(1999);
```

Comment:
Would it be an alternative to use the this:
```
import jdk.test.lib.Utils;
...
Random random = Utils.getRandomInstance();
```
This has a random seed, but it is always printed in the output and can be set via a test-flag.

Reply:
Changed.